src/bundle/Resources/public/js/scripts/admin.contenttype.edit.js

```diff
@@ -406,16 +436,11 @@
             if (sourceContainer.isEqualNode(targetContainer)) {
                 reorderFields({ targetContainer, draggedItemPosition });
             } else {
-                createFieldDefinitionNode(fieldRendered, { targetContainer, draggedItemPosition });
```

Comment:
I moved this function inside `addField` as it was called always just before `addField`, that's also why `addField` got new argument `fieldRendered`